### ISSUE TITLE
API stops working after enabling open tracing

### ISSUE THREAD

**Issue by maciejwojciechowski** [TykTechnologies/tyk#2685]
**Branch/Environment/Version**
- Branch/Version: release-2.9
- Environment: on-prem

**Describe the bug**
After enabling opentracing my 'auth Token' API stops working returning me error 
```
400 "error": "Session state is missing or unset! Please make sure that auth headers are properly applied" 
```
If tracing is disabled it works fine.

**Reproduction steps**

1. create auth token API
2. create key
3. send some traffic to see it is working fine
4. Enable tracing by adding this to your tyk.conf file
```
"tracing": {
        "enabled": true,
        "name": "jaeger",
        "options": {
        "baggage_restrictions": null,
        "disabled": false,
        "headers": null,
        "reporter": {
            "BufferFlushInterval": 0,
            "collectorEndpoint": "",
            "localAgentHostPort": "localhost:6831",
            "logSpans": true,
            "password": "",
            "queueSize": 0,
            "user": ""
        },
        "rpc_metrics": false,
        "sampler": {
            "maxOperations": 0,
            "param": 1,
            "samplingRefreshInterval": 0,
            "samplingServerURL": "",
            "type": "const"
        },
        "serviceName": "tyk-gateway",
        "tags": null,
        "throttler": null
        }
    }
```
5. Try to send traffic again.

**Actual behavior**
API stops working throwing 400 error

**Expected behavior**
API should work correctly

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/39672152/69422539-39ec6980-0d24-11ea-9f87-55ae09cd0989.png)

**Comment by maciejwojciechowski** [comment]
verified

**Comment by gathuku** [comment]
I can configured Jaeger using tyk-pro-docker-demo . But the gateway service is not created in Jaeger  UI. 

In tyk config
```
"tracing": {
      "enabled": true,
      "name": "jaeger",
      "options": {
        "baggage_restrictions": null,
        "disabled": false,
        "headers": null,
        "reporter": {
          "BufferFlushInterval": 0,
          "collectorEndpoint": "",
          "localAgentHostPort": "jaeger:6831",
          "logSpans": true,
          "password": "",
          "queueSize": 0,
          "user": ""
        },
        "rpc_metrics": false,
        "sampler": {
          "maxOperations": 0,
          "param": 1,
          "samplingRefreshInterval": 0,
          "samplingServerURL": "",
          "type": "const"
        },
        "serviceName": "tyk-gateway",
        "tags": null,
        "throttler": null
      }
    }
```
Jaeger docker-compose service 
```
  jaeger:
    image: jaegertracing/all-in-one:1.16
    environment:
      - COLLECTOR_ZIPKIN_HTTP_PORT=9411
    ports:
      - "5775:5775/udp"
      - "6831:6831/udp"
      - "6832:6832/udp"
      - "5778:5778"
      - "16686:16686"
      - "14268:14268"
      - "9411:9411"
    networks:
      - tyk
```

Am using latest tyk images. Any help?
![Screenshot from 2020-01-23 17-00-43](https://user-images.githubusercontent.com/26896865/72990866-f84e6c80-3e01-11ea-8f03-72d704e6e314.png)

**Comment by gathuku** [comment]
@gernest @buger

**Comment by gernest** [comment]
@gathuku did you try making api call?  traces will be recorded after making api calls not when tyk is initialised.

**Comment by gathuku** [comment]
It has started reporting but now getting bellow error
```
 time="Jan 24 07:26:04" level=error msg="http: proxy error: dial tcp: lookup jsonplaceholder.typicode.com on 127.0.0.11:53: server misbehaving" api_id=69713144fac641db5a155090ea7765c0 org_id=5dee117eab2ce40001b5e813 prefix=proxy server_name=jsonplaceholder.typicode.com user_id="****ce8e" user_ip=172.29.0.1 user_name=

```

**Comment by gernest** [comment]
@gathuku it seems you are having issue with target url, make sure you can talk to your api with curl first you are trying to reach `jsonplaceholder.typicode.co` that resolves to  ` 127.0.0.11:53` probably dns stuff. They have nothing to do with tyk .

**Comment by gathuku** [comment]
@gernest  I restarted and it worked.
